bin/prepare_environment.sh

```diff
@@ -16,41 +18,44 @@ else
     exit 1
 fi
 
+echo ""
+
 ## Clean up previous installations
-directories=("status-backend" "status-frontend" "node-red-status" "mysql" "collector-events")
+directories=("status-backend" "status-frontend" "node-red-status" "collector-events" ".env")
```

Comment:
.env file is also part of our script and it was missing here, so we remove it as well during setup now.